styleguide/components/00-settings/_colors.scss

```diff
@@ -204,6 +203,7 @@ $themes: (
     'color-box-shadow': $box-shadow-primary color('purple', -4),
     'color-primary-accolade': svg-as-background('accolade', color('purple', -4)),
     'color-primary-accolade-stroke': svg-as-background('accolade-stroke', color('cyan-wcag', -4)),
+    'color-primary-accolade-stroke2': svg-as-background('accolade-stroke', color('purple', 1)),
```

Comment:
Hmzs, maybe we need to rename this as a secondary accolade? What makes this one also primary?

Reply:
@Jeroen005 Because it's based on the same base color (cyan).

Reply:
No they're both a tint of cyan.

Reply:
But not in the subtheme 🤔?

Reply:
You're right. I've updated the names.